Implementation/Tabs with icon and svg/javascript/master.js

```diff
@@ -0,0 +1,13 @@
+let swapItems = document.querySelectorAll('.swp');
+swapItems.forEach((item, i) => {
+  item.addEventListener('mouseover', () => {
+    item.children[0].style.display="block";
+  });
+  item.addEventListener('mouseout', () => {
+    item.children[0].style.display="none";
+  });
+  item.addEventListener('click',() => {
+    document.querySelector('.swp-active').classList.remove("swp-active");
+    item.classList.add("swp-active");
```

Comment:
This looks good but more aesthetic feel would be by making it scroll

Reply:
Could you please explain it. A gif would help a lot.
I didnt quite understand it.

Reply:
Whenever we change the active tab it should move its way throught the page.